packages/ignition/src/app/robots.ts

```diff
@@ -6,9 +6,9 @@ const robots = (): MetadataRoute.Robots => {
     rules: {
       userAgent: "*",
       allow: "/",
-      disallow: "/api/"
+      disallow: ["/api/", "/_next/"]
     },
-    sitemap: `${serverEnv.NEXT_PUBLIC_APP_URL}/sitemap.xml`
+    sitemap: `${serverEnv.NEXT_PUBLIC_APP_URL}/sitemap_index.xml`
```

Comment:
Do we really need rename this? I'm afraid we're going to update few places and tools because of that.

Reply:
Yes we need. It is a standard. The only place I remember right now is Google Search Console.

Reply:
ok!